src/PostgresWebsockets.hs

```diff
@@ -81,11 +81,11 @@ wsApp getTime dbChannel secret pool multi pendingConn =
               Nothing -> pure ()
 
             when (hasRead mode) $
-              onMessage multi ch $ WS.sendTextData conn . B.payload
+              forM_ chs $ flip (onMessage multi) $ WS.sendTextData conn . B.payload
 
             when (hasWrite mode) $
               let sendNotifications = void . H.notifyPool pool dbChannel . toS
-              in notifySession validClaims (toS ch) conn getTime sendNotifications
+              in notifySession validClaims conn getTime sendNotifications $ toS first chs --temp just use first here, not sure how we will handle this in the case we only have a token with mutliple channels
```

Comment:
the part `toS first chs` won't type check. You probablt want to use something like `head`.
Although if you want some temporary code just to compile and run tests you should use `undefined`.

Reply:
in a complete version of this, how do you want to handle a write request for 

how do you want to handle the scenario where a a user opens a write allowed connection with multiple channels (using just a token), what channel do we say they write to?

Reply:
I'd say we should keep the behaviour consistent with the read, so in that case the write would deliver the same message to all channels